src/ui/state/manager.rs

```diff
@@ -151,25 +160,44 @@ impl From<ManagerHomeState> for Box<dyn State> {
     }
 }
 
+#[derive(Debug)]
+enum ManagerSendStep {
+    SelectOutputs(ManagerSelectOutputsView),
+    SelectInputs(ManagerSelectInputsView),
+    SelectFee(ManagerSelectFeeView),
+    Sign {
+        signer: SignState,
+        view: ManagerSignView,
+    },
+    Success(ManagerSpendTransactionCreatedView),
+}
+
 #[derive(Debug)]
 pub struct ManagerSendState {
     revaultd: Arc<RevaultD>,
-    view: ManagerSendView,
 
     warning: Option<Error>,
 
     vaults: Vec<ManagerSendInput>,
     outputs: Vec<ManagerSendOutput>,
+    feerate: u8,
+    psbt: Option<(Psbt, u8)>,
+    processing: bool,
+
+    step: ManagerSendStep,
 }
 
 impl ManagerSendState {
     pub fn new(revaultd: Arc<RevaultD>) -> Self {
         ManagerSendState {
             revaultd,
-            view: ManagerSendView::new(),
+            step: ManagerSendStep::SelectOutputs(ManagerSelectOutputsView::new()),
             warning: None,
             vaults: Vec::new(),
             outputs: vec![ManagerSendOutput::new()],
+            feerate: 20,
```

Comment:
Why is it hardcoded to 20 ? Also it seems to  be part of the `psbt` tuple as well